packages/form/src/component/number/CurrencyInput.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { Story } from "@storybook/react";
+
+import { CurrencyInput, CurrencyInputProps } from "./CurrencyInput";
+
+export default {
+  title: "Form/Component/CurrencyInput",
+  component: CurrencyInput,
+};
+
+const Template: Story<CurrencyInputProps> = (args) => {
+  return <CurrencyInput {...args} />;
+};
+
+export const Standard = Template.bind({});
+Standard.args = {};
```

Comment:
All stories except the last are failing for me with the following error: Currency must be provided ot CurrencyInput!